example/provider-local/garden/base/cloudprofile.yaml

```
@@ -28,6 +28,7 @@ spec:
     - version: 1.0.0
       cri:
       - name: containerd
+      - name: docker
```

Can you add the following comment to avoid confusion?

```suggestion
      # provider-local image doesn't contain a full docker runtime but includes nerdctl imitating the docker CLI for fulling gardener's bootstrap needs:
      # see https://github.com/gardener/machine-controller-manager-provider-local/blob/f2c93198c794afc4e8e742a26026584ecce9aadf/node/Dockerfile#L9-L20
      # this can be removed as soon as https://github.com/gardener/gardener/issues/4673 is resolved
      - name: docker
```